testbed/src/main/java/org/spdx/testbed/Main.java

```diff
@@ -4,71 +4,88 @@
 
 package org.spdx.testbed;
 
-import org.apache.commons.cli.*;
+import org.apache.commons.cli.CommandLine;
+import org.apache.commons.cli.DefaultParser;
+import org.apache.commons.cli.HelpFormatter;
+import org.apache.commons.cli.Option;
+import org.apache.commons.cli.Options;
+import org.apache.commons.cli.ParseException;
 import org.spdx.library.InvalidSPDXAnalysisException;
-import org.spdx.testbed.generationTestCases.*;
+import org.spdx.testbed.util.TestCaseSelector;
 import org.spdx.tools.InvalidFileNameException;
 
 import java.io.IOException;
+import java.util.List;
 
 public class Main {
-    public static void main(String[] args) throws IOException, InvalidSPDXAnalysisException, InvalidFileNameException {
+    public static void main(String[] args) throws IOException, InvalidSPDXAnalysisException,
+            InvalidFileNameException {
         var options = new Options();
-        options.addOption(Option.builder("t").longOpt("test_case").desc("For possible values see some website.").hasArg().argName("TESTCASE").required().build());
-        options.addOption(Option.builder("f").longOpt("input_files").desc("The files to be processed").hasArgs().argName("FILES").required().build());
-        options.addOption(Option.builder("h").longOpt("help").desc("Display usage").required(false).build());
+        options.addOption(Option.builder("t").longOpt("test_cases")
+                .desc("For possible values see the readme. At least one of -c or -t has to be " +
```

Comment:
It says "at least one" but when I provide
```
-c generation -t generationMinimalTest
```
zero testCases will be selected

Reply:
Aaahhhh... bug! Good catch - and I just realized that this part of the logic (i.e.: the combination of names and categories) is completely untested :flushed:

Reply:
It fails because I am comparing separate instances of `TestCase` by object reference when I should really be comparing by name...

Reply:
Fixed